satellite/console/service.go

```diff
@@ -168,6 +168,141 @@ func (s *Service) CreateUser(ctx context.Context, user CreateUser, tokenSecret R
 	return u, nil
 }
 
+// AddNewPaymentMethod - is a method to add new payment method for project
```

Comment:
For all comments, the idiomatic Go way is to write it as a sentence where the method name is part of it. for example:
```

Reply:
I made it to look consistent. Like `GenerateActivationToken`, `GeneratePasswordRecoveryToken`, `ActivateAccount` and others.

Reply:
I can rename all occurrences at service, or just rename mine